contracts/modules/assessment/Assessment.sol

```diff
@@ -170,26 +176,30 @@ contract Assessment is IAssessment, RegistryAware, Multicall {
   function setAssessmentDataForProductTypes(
     uint[] calldata productTypeIds,
     uint cooldownPeriod,
+    uint payoutRedemptionPeriod,
     uint groupId
   ) override external onlyContracts(C_GOVERNOR) {
+
     require(groupId > 0 && groupId <= _groupCount, InvalidGroupId());
 
     uint length = productTypeIds.length;
     for (uint i = 0; i < length; i++) {
       _assessmentData[productTypeIds[i]] = AssessmentData({
         assessingGroupId: groupId.toUint16(),
-        cooldownPeriod: cooldownPeriod.toUint32()
+        cooldownPeriod: cooldownPeriod.toUint32(),
+        payoutRedemptionPeriod: payoutRedemptionPeriod.toUint32()
       });
     }
 
-    emit AssessmentDataForProductTypesSet(productTypeIds, cooldownPeriod, groupId);
+    emit AssessmentDataForProductTypesSet(productTypeIds, cooldownPeriod, payoutRedemptionPeriod, groupId);
```

Comment:
That's a very cool event you've got there. It would be a shame if someone asked you how would you query this event for a specific product type.

Reply:
good catch!

Reply:
I've indexed both productTypeId and groupId:

```solidity
  event AssessmentDataForProductTypeSet(
    uint indexed productTypeId,
    uint indexed groupId,
    uint cooldownPeriod,
    uint payoutRedemptionPeriod
  );
```